guides/common/modules/ref_supported-clients-in-registration.adoc

```diff
@@ -0,0 +1,68 @@
+[id="supported-clients-in-registration_{context}"]
+= Supported Clients in Registration
+
+{Project} supports the following operating systems and architectures for registration.
+
+Supported Host Operating Systems::
+The hosts can use the following OS versions:
+
+ifndef::satellite[]
+* AlmaLinux
+* Amazon Linux
+* CentOS
+endif::[]
+ifndef::satellite[]
+* Debian
+ifndef::orcharhino[]
+* Fedora
+* FreeBSD
+* OpenSUSE
+endif::[]
+* Oracle Linux
+endif::[]
+* {EL} 9, 8, 7
+ifdef::satellite[]
+* {EL} 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
+endif::[]
+ifndef::satellite[]
+* {SLES}
+ifndef::orcharhino[]
+* Solaris
+endif::[]
+* Ubuntu
+endif::[]
+
+Supported Host Architectures::
+The hosts can use the following architectures:
```

Comment:
@ekohl @maximiliankolb @stejskalleos Does this mean that only these architectures can be registered? Or does it mean that Project provides client tooling only for these architectures?

Reply:
I'm trying to ask what does it mean when we say that the host architectures are _supported in registration_? What exactly is supported?

Reply:
> What exactly is supported?

Let's start here, because supported needs to be defined and it's good to repeat it from time to time. In Red Hat supported means you can call Red Hat and get it fixed.

In Foreman it's more ambiguous because there is no customer support, only community support. Does supported means it can be made to work? Or does it mean it's provided out of the box?

You can consider side stepping the issue and using something like `tested` or `verified`. What you can also do (and I'm now leaning here) is add an additional note in upstream explaining that the supported combinations is just what is known to work, but also explain that it's just a template for a shell script (link to [the global registration template](https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/registration/global_registration.erb)) which can be modified. Also encourage submitting those changes back upstream.